src/openrct2-ui/windows/GuestList.cpp

```diff
@@ -857,7 +857,7 @@ static void window_guest_list_scrollpaint(rct_window* w, rct_drawpixelinfo* dpi,
                     std::memcpy(
                         gCommonFormatArgs, _window_guest_list_groups_arguments[i].args,
                         std::min(sizeof(gCommonFormatArgs), sizeof(_window_guest_list_groups_arguments[i].args)));
-                    gfx_draw_string_left_clipped(dpi, format, gCommonFormatArgs, COLOUR_BLACK, { 0, y }, 414);
+                    DrawTextEllipsised(dpi, { 0, y }, 414, format, gCommonFormatArgs, COLOUR_BLACK);
```

Comment:
@duncanspumpkin OK, then tell me how to do this line, for example.

Reply:
```Formatter ft(_window_guest_list_groups_arguments[i].args);
DrawTextEllipsised(dpi, { 0, y }, 414, format, ft, COLOUR_BLACK);
```